MockKitUI/MockPumpManager+UI.swift

```diff
@@ -11,8 +11,13 @@ import LoopKit
 import LoopKitUI
 import MockKit
 
+private class FrameworkBundle {
```

Comment:
nit: Using this class hack (I can call it a hack since the pattern in Loop is my fault. ;) to implement a helper for this single use is not worth it here, in my opinion. I think it's a little cleaner and clearer the way it was.

Reply:
Ok... but IMHO reaching up to `MockPumpManagerSettingsViewController.self` from here seemed like a less clean way to do it.  I actually thought your "hack" was a good idea :).